examples/celery-rabbitmq/flower-controller.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1beta3
+kind: ReplicationController
+metadata:
+  labels:
+    name: flower
+  name: flower-controller
+spec:
+  replicas: 1
+  selector:
+    name: flower
+  template:
+    metadata:
+      labels:
+        app: taskQueue
+        name: flower
```

Comment:
I'd use "component" as the label key rather than "name", to better distinguish it from "app". We've also received feedback that using "name" as a label key is confusing, in general (#3394).